apps/aecore/test/aec_mining_tests.erl

```diff
@@ -114,7 +115,8 @@ generate_valid_test_data(TopBlock, Tries) ->
                             [{TopBlock, aec_trees:new()}], ?TEST_PUB),
     HeaderBin = aec_headers:serialize_to_binary(aec_blocks:to_header(BlockCandidate)),
     Target = aec_blocks:target(BlockCandidate),
-    case ?TEST_MODULE:generate(HeaderBin, Target, Nonce, 0) of
```

Comment:
🤔

Reply:
aec_mining PR should have changed this, but this is code that is not run normally...